x-pack/plugins/ingest_pipelines/public/application/components/pipeline_form/schema.ts

```diff
@@ -8,12 +8,25 @@
 import { i18n } from '@kbn/i18n';
 import { FormSchema, FIELD_TYPES, fieldValidators, fieldFormatters } from '../../../shared_imports';
 
-import { PipelineForm } from './types';
-
-const { emptyField } = fieldValidators;
+const { emptyField, isJsonField } = fieldValidators;
 const { toInt } = fieldFormatters;
 
-export const pipelineFormSchema: FormSchema<PipelineForm> = {
+const stringifyJson = (json: { [key: string]: any }): string =>
```

Comment:
I see theres a similar implementation for these two functions in `x-pack/plugins/ingest_pipelines/public/application/lib/utils.ts `. Is there any chance we could reuse this or perhaps refactor the original functions in order to support both cases?

Reply:
Thanks for this suggestion! I didn't spot these very similar functions in this file. I refactored them so that they can be used in this case as well. Let me know if you think they can be changed in a better way.